src/DataProtection/DataProtection/test/KeyManagement/XmlKeyManagerTests.cs

```diff
@@ -51,20 +51,24 @@ public void Ctor_WithoutEncryptorOrRepository_UsesFallback()
     }
 
     [Fact]
-    public void Ctor_WithEncryptorButNoRepository_IgnoresFallback_FailsWithServiceNotFound()
+    public void Ctor_WithEncryptorButNoRepository_UsesFallback()
```

Comment:
Can we add a test for the public API mentioned in the issue?

`DataProtectionProvider.Create(string applicationName, X509Certificate2 certificate)`